src/link-rewriter.js

```diff
@@ -39,12 +38,14 @@ module.exports = class LinkRewriter {
 				continue;
 			}
 
-			console.log(`Found relative link: '${href.value}' in '${item.descriptionFileName || item.fileName}`);
+			let targetPath = href.value;
+			if (targetPath.startsWith('.')) {
```

Comment:
Are there resources linked using `./path/filename.ext`? Seems odd for HTML.

Reply:
It might be odd, but I prefer using ./index.html over index.html.  For whatever reason, when I see index.html it takes me half a second to convince myself that it's relative and not absolute... so write ./index.html to be explicit.